eng/Generate.ps1

```diff
@@ -238,6 +238,22 @@ if (!($Exclude -contains "Samples"))
     }
 }
 
+# Sample for cadl project
+$cadlSampleProjectName = 
+    'AnomalyDetector'
+
+if (!($Exclude -contains "Samples"))
+{
+    foreach ($projectName in $cadlSampleProjectName)
+    {
+        $projectDirectory = Join-Path $repoRoot 'samples' $projectName
+        $cadlMain = Join-Path $projectDirectory "main.cadl"
+        $cadlClient = Join-Path $projectDirectory "main.cadl"
+        $mainCadlFile = If (Test-Path "$cadlClient") { Resolve-Path "$cadlClient" } Else { Resolve-Path "$cadlMain"}
+        Add-Cadl $projectName $projectDirectory $mainCadlFile "--option @azure-tools/cadl-csharp.generateConvenienceAPI=true --option @azure-tools/cadl-csharp.unreferenced-types-handling=keepAll"
```

Comment:
better to provide some prompt messages like `Generating CADL sample project ${projectName}...`

Reply:
Here is just to add the sample project as candidate, not to generate sdk.  It will display similar message when generate sdk for this project. So it is not necessary to provide message here.